requirements-base.txt

```diff
@@ -61,6 +61,7 @@ requests>=2.25.1
 rfc3339-validator>=0.1.2
 rfc3986-validator>=0.1.1
 # [end] jsonschema format validators
+selenium>=4.16.0
```

Comment:
I am very concerned about trying to ship selenium and a headless chrome to production

this is not going to work as written fwiw.  I also think we should pursue a different way to do this since it will ~3x the size of our production image by adding chromium and xvfb

Reply:
can you rope in security as well -- I'm worried about the access controls on such an endpoint as well -- cc @mdtro